test/staleIfError.js

```diff
@@ -102,8 +110,11 @@ describe('Stale-If-Error', () => {
         .asResponse();
     }
 
-    await doesNotReject(requestWithCacheAndNextMiddleware, /Error starting da cache/);
-    assert.strictEqual(called, true, 'Expected the next middleware to be called');
+    try {
+      await requestWithCacheAndNextMiddleware();
+    } catch (error) {
+      throw error;
```

Comment:
Need to fail test in case we don't get into the `catch` as above

Reply:
Hi Nick, we're not expecting an error to be thrown in this case.

Reply:
Oh I see, yes. I don't think this needs to be in a `try catch` then

Reply:
done